.travis.yml

```diff
@@ -16,6 +16,8 @@ node_js:
   - "6"
```

Comment:
I think we can remove 6 and 8, since they are EOL.

Reply:
Yeah, but that’s a semver-major change and I’d like to avoid that at this point.

Reply:
Since Node.js 13.6.0 comes out, maybe add 13 too.

Reply:
Unfortunately, tests don’t pass on 13 yet

Reply:
Why the stack pointed to Node.js source code on Node.js 13 ?
```console
  1) Callback
       "after each" hook for "should create a C function pointer from a JS function":
     TypeError: Cannot read property 'call' of undefined
      at processImmediate (internal/timers.js:456:21)
```

Reply:
Ok, find the cause 
https://github.com/node-ffi-napi/node-ffi-napi/blob/18a2a9f0e93e83cc3ee710b920237ebe14da04d1/test/variadic.js#L9

Reply:
Okay, this is macOS error. On linux it's another error.